playbooks/ansible/roles/sit.cephfs/templates/ceph.smb.cluster.yml.j2

```diff
@@ -42,3 +42,6 @@ resources:
     {%- for addr in ctdb_network_public_interfaces +%}
     - address: {{ addr }}/{{ ctdb_network_public_interface_subnet_mask }}
     {%- endfor +%}
+  custom_smb_global_options:
+    "_allow_customization": "i-take-responsibility-for-all-samba-configuration-errors"
+    "posix locking": "no"
```

Comment:
Is this something that should also be done in the smb mgr module ? if we are going to support durable handles there, the module should take care of it, right ?

Reply:
This configuration change (as expected [here](https://docs.ceph.com/en/latest/mgr/smb/)) is only for smb mgr module based clusters. The default variant make use of _[smb.conf.j2](https://github.com/samba-in-kubernetes/sit-environment/blob/main/playbooks/ansible/roles/samba.setup/templates/smb.conf.j2)_ where `posix locking` is already disabled.

Reply:
Ok. That's fine. Thanks!